editoast/src/models/paced_train.rs

```diff
@@ -3,7 +3,7 @@ use chrono::Duration as ChronoDuration;
 use chrono::Utc;
 use editoast_derive::Model;
 use editoast_schemas::paced_train::Paced;
-use editoast_schemas::paced_train::PacedTrainBase;
+use editoast_schemas::paced_train::PacedTrain as PacedTrainSchema;
```

Comment:
I'd argue that using the fully qualified path enhances readability in this context